certified-connectors/Formstack Forms/apiDefinition.swagger.json

```diff
@@ -0,0 +1,243 @@
+{
+  "swagger": "2.0",
+  "info": {
+    "title": "Formstack",
```

Comment:
Just to double-check, should the title be `Formstack` or `Formstack Forms`?

Reply:
@dougelliott Nice catch - I will change and update the PR with a new commit.  Anything else we see being an issue?

Reply:
@bbillstrFS With Srikanth's approval coming in, that should be it :)

Reply:
@dougelliott Confirmed with Gabe that we want Formstack Forms here - is the title in swagger necessary, or does the name we specify when we submit to ISV Studio the main driver for connector name?

Reply:
@bbillstrFS It's actually the other way around, the title in the swagger is what matters most, as that is what will be used across the product. The name in ISV Studio should ideally match, but it has no bearing on what ends up getting deployed. So if you'd like Formstack Forms, go ahead and change this field as such and you'll be good to go.

Reply:
@dougelliott Fantastic - thanks so much for the quick replies here!  This change has been made and latest commit should be tied to this PR.

Reply:
@bbillstrFS Awesome, approved. Go ahead and submit in ISV Studio and we'll get things started.